dolphinscheduler-python/pom.xml

```diff
@@ -104,5 +104,67 @@
                 </plugins>
             </build>
         </profile>
+        <profile>
+            <id>release</id>
+            <build>
+                <plugins>
+                    <plugin>
+                        <groupId>org.codehaus.mojo</groupId>
+                        <artifactId>exec-maven-plugin</artifactId>
```

Comment:
It is odd, every time I build the package with command `mvn -U clean package -Prelease`. It would run `docker-build`, and I think it is because the plugins `exec-maven-plugin` in the parent `pom.xml` have two executions (`docker-build` and `docker-push`). Is there any way could just run children's executions without parents? @kezhenxu94 @CalvinKirs

Reply:
I see, this is an issue.

Let me fix this

Reply:
I pushed a commit to address this, @zhongjiajie can you confirm?